constants/data.ts

```diff
@@ -2,346 +2,395 @@ import { ProjectType } from 'entities/project'
 
 export const DEPARTMENTS: ProjectType[] = [
   {
-    "name": "Marketing Department",
-    "page": "department",
-    "description": "The Marketing Department is dedicated to amplifying the values of BanklessDAO, accomplishing this by delivering creative Web3 marketing solutions that effectively promote BanklessDAO initiatives, partners, and clients to their target audience with precision. Despite not yet adapting our own values, we remain aligned with those of the DAO.",
-    "image": "/images/department/marketing-department-de80a554.png"
+    name: 'Marketing Department',
+    page: 'department',
+    description:
+      'The Marketing Department is dedicated to amplifying the values of BanklessDAO, accomplishing this by delivering creative Web3 marketing solutions that effectively promote BanklessDAO initiatives, partners, and clients to their target audience with precision. Despite not yet adapting our own values, we remain aligned with those of the DAO.',
+    image: '/images/department/marketing-department-de80a554.png',
```

Comment:
@didierkrux Will re-formatting this file break the scripts for syncing with Notion?